### ISSUE TITLE
Would be cool if there was an option to ignore Git paths

### ISSUE THREAD

**Issue by kvz** [tjunnone/npm-check-updates#64]
Thanks for npm-check-updates, I think it's really helpful and a big time saver if you run many projects :+1:

A small suggestion to make it even better - I ran `npm-check-updates --upgrade` on a `package.json` that contained:

```javascript
    "librato-metrics": "git://github.com/holidayextras/node-librato-metrics.git#1ac0dc99fb",
```

Which was then changed to:

```javascript
    "librato-metrics": "git://github.com/holidaye0.0",
```

The same thing happened for a few other modules. I realize detecting latest versions for repos gets hairy quickly and I would vote against that.

But just (an option for) ignoring anything that starts with `git://` would be great here already.

My version:

```bash
$ npm-check-updates --version
1.5.1
```

**Comment by metaraine** [comment]
Thanks! That is clearly better behavior, and has been implemented in the soon-to-be-released v2.0.0. You can get this behavior in v2.0.0-alpha.7 currently:

`npm install -g npm-check-updates@unstable`

**Comment by kvz** [comment]
Works like a charm, thanks!
